include/common/int128.h

```diff
@@ -14,17 +14,20 @@
 #pragma once
 
 #if defined(__x86_64__) || defined(__aarch64__)
+#include <ostream>
```

Comment:
Looks like the `<ostream> is contained in both `if` and `else` conditions. Maybe we can move this include out?